engine/app/components/citizens_advice_components/footer.rb

```diff
@@ -50,6 +52,16 @@ def feedback_url_deprecation
       )
     end
 
+    def legal_summary
+      legal_summary_slot.presence || legal_summary_fallback.presence
+    end
+
+    def legal_summary_fallback
```

Comment:
I've added this to avoid making a breaking change by making the legal summary slot required. I've not added a depreciation warning because in most of the cases, we want to use the default legal summary and only add a slot when a custom text is needed